magtogoek/app.py

```diff
@@ -210,6 +210,10 @@ def config_adcp(
               help="""year when the adcp sampling started. ex: `1970`""", required=True)
 @click.option("-T", "--platform_type", type=click.Choice(["buoy", "mooring", "ship"]),
               help="Used for Proper BODC variables names", default="buoy")
+@click.option("--show-fig/--hide-fig", "showfig",
```

Comment:
As discused, I think an option to tell magtogoek that the user is remotly using it remotly, thus not having a graphical interface, is the way to go. In this case, if the option `make_fig` in the config file is True (and not a path),  magtogoek should change that value for the default_path for output. If `make_fig` is a path, then the figure will saved anyway. In both case this should supress the figure being plotted. The option `no-make-fig` would overwrite both plotting and saving of the figures. The option for remote usage of magtogoek should also be added to the quick command.